bin/bossSwitch.py

```diff
@@ -0,0 +1,210 @@
+    subprocess.call('./bastion.py ' + args.vpc + ' vault-unseal',shell=True)
+    print("Importing vault content")
+    subprocess.call('./bastion.py ' + args.vpc + ' vault-import < ../config/vault_export.json', shell=True)
+    print(bcolors.WARNING + "Successful import" + bcolors.ENDC)    
+
+    #Start endpoint and activities instances
+    print("Starting endpoint, and activities...")
+    client.update_auto_scaling_group(AutoScalingGroupName=endpoint, MinSize = endpointD[0] , MaxSize = endpointD[1] , DesiredCapacity = endpointD[2])
+    client.resume_processes(AutoScalingGroupName=endpoint,ScalingProcesses=['HealthCheck'])
+    
+    client.update_auto_scaling_group(AutoScalingGroupName=activities, MinSize = activitiesD[0] , MaxSize = activitiesD[1] , DesiredCapacity = activitiesD[2])
+    client.resume_processes(AutoScalingGroupName=activities,ScalingProcesses=['HealthCheck'])
+
+    print(bcolors.OKGREEN + "TheBoss is on" + bcolors.ENDC)
+
+
+def stopInstances():
+    """
+        Method used to stop currently running instances
+    """
+    #Save current ASG descriptions:
+    print("Saving current auto scaling group configuration...")
+    response = client.describe_auto_scaling_groups(AutoScalingGroupNames=[endpoint,activities,vaultg])
+    save_obj(response,'ASGdescriptions')
+    
+    #Export vault content:
+    print("Exporting vault content...")
+    subprocess.call('./bastion.py ' + args.vpc + ' vault-export > ../config/vault_export.json', shell=True)
+    print(bcolors.WARNING + "Successful export" + bcolors.ENDC)
+
+    #Switch off:
+    print("Stopping all Instances...")
+    client.update_auto_scaling_group(AutoScalingGroupName=endpoint, MinSize = 0 , MaxSize = 0 , DesiredCapacity = 0)
+    client.suspend_processes(AutoScalingGroupName=endpoint,ScalingProcesses=['HealthCheck'])
+
+    client.update_auto_scaling_group(AutoScalingGroupName=activities, MinSize = 0 , MaxSize = 0 , DesiredCapacity = 0)
+    client.suspend_processes(AutoScalingGroupName=activities,ScalingProcesses=['HealthCheck'])
+
+    client.update_auto_scaling_group(AutoScalingGroupName=vaultg, MinSize = 0 , MaxSize = 0 , DesiredCapacity = 0)
+    client.suspend_processes(AutoScalingGroupName=vaultg,ScalingProcesses=['HealthCheck'])
+
+    print(bcolors.FAIL + "TheBoss is off" + bcolors.ENDC)
+
+def save_obj(obj, name ):
+    """
+        Method to save objects as .pkl files
+
+        Args:
+            obj : The object that will be saved
+            name : The .pkl file name under which the object will be saved
+    """
+    with open('../'+ name + '.pkl', 'wb') as f:
+        pickle.dump(obj, f, pickle.HIGHEST_PROTOCOL)
+
+def load_obj(name):
+    """
+        Method to load saved objects in .pkl file
+
+        Args:
+            name : The .pkl file name to open
+        
+        Returns:
+            object saved within .pkl file
+    """
+    with open('../' + name + '.pkl', 'rb') as f:
+        return pickle.load(f)
+
+class bcolors:
```

Comment:
Is there a Python library that would help with rendering colored output? Just wondering how this holds up for weird terminals.